core/domain/stats_jobs_one_off.py

```diff
@@ -429,7 +443,9 @@ def reduce(exp_id, stringified_values):
                 dest_states = [
                     answer_group.outcome.dest
                     for answer_group in init_state.interaction.answer_groups]
-                dest_states.append(init_state.interaction.default_outcome.dest)
+                if init_state.interaction.default_outcome:
```

Comment:
if init_state.interaction.default_outcome is not None:

Reply:
Done.